tests/kafka/kafka.go

```diff
@@ -14,94 +14,157 @@
 package main
```

Comment:
I prefer to keep testing the old way without partition(this is more important) if you want you should add another test instead of changing the previous test.